src/components/atoms/ReviewCardCarousel/ReviewCard.module.scss

```diff
@@ -0,0 +1,35 @@
+.card-container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  justify-content: center;
+  width: 100%;
+}
+
+.card-img {
+  border: 5px solid #c33; /* Shortened the hex color */
```

Comment:
Could you create color variables for our review card and use them here? You can check other card implementations. It will enable us to easily change colors in the future.